rs/ic_os/config/src/types.rs

```diff
@@ -87,7 +87,7 @@ pub struct ICOSSettings {
     /// This file must be a text file containing the public key of the NNS to be used.
     pub nns_public_key_path: PathBuf,
     /// The URL (HTTP) of the NNS node(s).
-    pub nns_urls: Vec<Url>,
+    pub nns_url: Vec<Url>,
```

Comment:
This is still a list right? Why the name change?

Reply:
The value that it populates in nns.conf is `nns_url`

So I could change all references to be nns_urls in the ic-os config up to the population of nns.conf in guestos, or I could leave it as nns_url.

I decided to go with nns_url because deployment.json just lists it as `url`, but I could go either way. But yes, this is still a list

Reply:
I'd prefer to move the other direction towards `nns_urls`, since we do use it as a list. This is especially hard to tell in places where we're actually packing it as a string with commas: https://github.com/dfinity/ic/blob/fcad095e73fa6ffe88fa5a6479a010f8b1b826b0/rs/orchestrator/src/registration.rs#L451

But I know a lot of this is in flux. If it helps to keep the naming consistent for now, that's alright, but the end state should definitely go back to plural, and hopefully clean up some of the string splitting we have to do now.

Reply:
Sounds good! I'll do this in a separate PR.